plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/DBReportMediatorPropertiesEditionComponent.java

```diff
@@ -667,9 +668,10 @@ public void updatePart(Notification msg) {
 	 * 
 	 */
 	public String getHelpContent(Object key, int kind) {
-		if (key == EsbViewsRepository.DBReportMediator.Statements.sqlStatements)
-			return "SQL Statements"; //$NON-NLS-1$
-		return super.getHelpContent(key, kind);
+//		if (key == EsbViewsRepository.DBReportMediator.Statements.sqlStatements)
```

Comment:
Remove commented lines.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804